### PR TITLE
Add progress bar

### PR DESCRIPTION
**Pull request by BeckyBlake**
Screenshot 2024-11-03 at 12 49 57 PM

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 **/.next
 **/.husky
 public/registration/*.svg
+public/registration/progress_bar/*.svg
```
Comment (J164) [inline .prettierignore]: Let's have prettier ignore the entire public/ directory

File: components/ProgressBar/ProgressBar.module.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        margin: auto;
+        background-color: #ddd;
+        border-radius: 3px;
+        margin-bottom: 20px;
+
+        .progressBarFill {
+            height: 100%;
+            background-color: #a19366;
+            border-radius: 3px;
+            transition: width 0.4s ease;
```
Comment (J164) [inline components/ProgressBar/ProgressBar.module.scss]: can we try ease-out and a longer animation for this? The current one seems a bit rushed

File: components/ProgressBar/ProgressBar.module.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            justify-content: center;
+            cursor: pointer;
+        }
+
+        .step {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+
+            img {
```
Comment (J164) [inline components/ProgressBar/ProgressBar.module.scss:54]: can we make the icons smaller when on mobile devices?

**Review by J164 — CHANGES_REQUESTED** [review]
This is a great start! I left a few comments to fix up before merging 🚀

**Review by J164 — CHANGES_REQUESTED** [review]
There seems to be a small alignment issue:
image

**Comment by BeckyBlake** [comment]
I've been having trouble with the moon and lightning bolt on the ends because if they're perfectly centered in their containers like the other symbols, then the bar peeks out on the ends.
Screenshot 2024-11-06 at 5 48 57 PM

 i used negative margins on the moon and lightning to make it so that the bar doesn't peek out but as screen size decreases the difference in spacing becomes more noticeable. i'm not sure if you would consider this to be a significant difference in spacing or if it's fine
Screenshot 2024-11-06 at 5 45 48 PM

**Comment by J164** [comment]
> I've been having trouble with the moon and lightning bolt on the ends because if they're perfectly centered in their containers like the other symbols, then the bar peeks out on the ends.
> 
> i used negative margins on the moon and lightning to make it so that the bar doesn't peek out but as screen size decreases the difference in spacing becomes more noticeable. i'm not sure if you would consider this to be a significant difference in spacing or if it's fine
 
This looks good for now. We can revisit if we have time before registration deadline

**Review by J164 — APPROVED** [review]
Awesome job on this! Looks great 🚀

**Pull request merged**